abs.yaml

```diff
@@ -0,0 +1,6 @@
+build_env_vars:
+  ANACONDA_ROCKET_ENABLE_PY313: yes
```

Comment:
```suggestion
```
I think this can be removed since we're skipping python 3.13 in the meta.yaml

Reply:
Unless we're not supposed to be skipping it. Then I would suggest removing the skip instead

Reply:
Good point! I'll get it removed via next push.